frontend/package.json

```diff
@@ -178,7 +178,9 @@
       "<rootDir>/src/widgets/Example.js",
       "<rootDir>/src/pages/RegionalDashboard/formatDateRange.js",
       "<rootDir>/src/pages/RegionalDashboard/constants.js",
-      "<rootDir>/src/pages/makecolors.js"
+      "<rootDir>/src/pages/makecolors.js",
+      "<rootDir>/src/NetworkContext.js",
+      "<rootDir>/src/hooks/helpers.js"
```

Comment:
Both network context and helpers are mainly if statements around low level browser APIs and how properties are named differently. It seemed unnecessary to test that